wake-gce.yml

```diff
@@ -9,7 +9,7 @@
         file: src=secrets.py dest=./secrets.py mode=0700
 
       - name: Wake instances
-        shell: "./gce.py --start {{ item }}"
+        shell: "./gce.py --start {{ item }} &"
```

Comment:
Would it be better to use [`async`](http://docs.ansible.com/ansible/playbooks_async.html) so that Ansible doesn't exit until they are all finished?